### DISCUSSION TITLE
Problem in reverifying academic status

### DISCUSSION THREAD

**Discussion by gautam8404** [community/community#61459] [GitHub Education]
When I first applied for GitHub student pack my university wasn't in list nor they had provided any student email so I got verified by submitting my student ID card

This year however for reverifying my academic status my university is in list and GitHub says I have to use e-mail provided by my university however my university have not provided any email, so like last time I entered my uni name and submitted my ID card but verification failed

![Screenshot_2023-07-21-19-29-50-59_e307a3f9df9f380ebaf106e1dc980bb6.jpg](https://github.com/community/community/assets/93448885/4cf730d0-26ab-48d5-834b-316f3dbb09e2)

My university have not provided me or anyone in my class an email, I asked my professor they said they do not provide students email what should I do

**Comment by nonvenomous** [comment]
Same issue here and we continue to get ghosted by github.
Here a bigger thread https://github.com/orgs/community/discussions/59360

**Reply by gautam8404** [reply]
Ye i even tried to contact through support, no response

**Comment by Zweistein2** [comment]
same here. No email provided by my university and my uploaded documents to proof my enrollment get rejected instantly.
